src/engine/shared/config_variables_insta.h

```diff
@@ -133,3 +133,10 @@ MACRO_CONFIG_INT(SvRoundStatsFormatDiscord, sv_round_stats_format_discord, 1, 0,
 MACRO_CONFIG_INT(SvRoundStatsFormatHttp, sv_round_stats_format_http, 4, 0, 4, CFGFLAG_SERVER, "0=csv 1=psv 2=ascii table 3=markdown table 4=json")
 MACRO_CONFIG_INT(SvRoundStatsFormatFile, sv_round_stats_format_file, 1, 0, 4, CFGFLAG_SERVER, "0=csv 1=psv 2=ascii table 3=markdown table 4=json")
 MACRO_CONFIG_INT(SvPrintRoundStats, sv_print_round_stats, 0, 0, 1, CFGFLAG_SERVER, "print top players in chat on round end")
+MACRO_CONFIG_INT(SvBallDestroyededOnDeathTile, sv_ball_destroyeded_on_death_tile, 0, 0, 1, CFGFLAG_SERVER, "Destroy the ball when it touches death tile (only foot)")
+MACRO_CONFIG_INT(SvBallBounceFriction, sv_ball_bounce_friction, 50, 0, 100000, CFGFLAG_SERVER, "The ball looses that much speed after a bounce (only foot)")
+MACRO_CONFIG_INT(SvBallExplode, sv_ball_explode, 0, 0, 1, CFGFLAG_SERVER, "Should the grenades explode (only foot)")
+MACRO_CONFIG_INT(SvBallSpawnDelay, sv_spawn_delay, 2000, 0, 100000, CFGFLAG_SERVER, "Spawn delay for players after a kill (only foot)")
```

Comment:
This should be removed we already have `sv_self_kill_respawn_delay_ms` working for all modes.